Tests/Magick.NET.Tests/Shared/Pixels/SafePixelCollectionTests/TheToByteArrayMethod.cs

```diff
@@ -50,6 +51,21 @@ public void ShouldReturnPixelsWhenAreaIsCorrecy()
                 }
             }
 
+            [TestMethod]
+            public void ShouldReturnPixelsWhenAreaAndMappingAreCorrect()
+            {
+                using (IMagickImage image = new MagickImage(Files.ImageMagickJPG))
+                {
+                    using (IPixelCollection pixels = image.GetPixels())
+                    {
+                        var values = pixels.ToByteArray(60, 60, 63, 58, PixelMapping.RGBA);
+                        int length = 63 * 58 * 4;
+
+                        Assert.AreEqual(length, values.Length);
+                    }
+                }
+            }
+
             [TestMethod]
             public void ShouldThrowExceptionWhenGeometryIsNull()
```

Comment:
Can you also add a test for `ShouldThrowExceptionWhenGeometryIsNullAndMappingIsEnum`?